config/config.sample.php

```diff
@@ -1428,6 +1428,14 @@
  */
 'metadata_max_filesize' => 256,
 
+/**
+ * Maximum file size for file conversion.
+ * If a file exceeds this size, the file will not be converted.
+ *
+ * Default: 100 megabytes
```

Comment:
```suggestion
 * Default: 100 MiB
```
I think you mean mibibytes (base 1024 and not 1000 like we use it everywhere else) and using the abbreviation is a bit more common.